src/lro/src/details.rs

```diff
@@ -149,12 +149,12 @@ where
     R: wkt::message::Message + serde::ser::Serialize + serde::de::DeserializeOwned,
 {
     if let Some(any) = op.response() {
-        return any.to_msg::<R>().map_err(Error::serde);
+        return any.to_msg::<R>().map_err(Error::deser);
     }
     if let Some(e) = op.error() {
         return Err(Error::service(gax::error::rpc::Status::from(e.clone())));
     }
-    Err(Error::serde("missing result in completed operation"))
+    Err(Error::deser("missing result in completed operation"))
```

Comment:
Hmm... this is some sort of service bug. I don't like `serde` or `deser`.

I think it would get its own error type. Maybe switch it to `other` temporarily to unblock this PR?

Reply:
Hmmm... grumble... Okay, but you need to come up with a name for that error kind.